v8/rest/template.ts

```diff
@@ -0,0 +1,34 @@
+import type { APIGuild, APITemplate } from '../payloads';
+
+/**
+ * https://github.com/discord/discord-api-docs/blob/master/docs/resources/Template.md#get-template--get-guildstemplatestemplatecodedocs_resources_templatetemplate-object
+ */
```

Comment:
`REST<Method>APIGuildTemplate` please 🙏

Reply:
And what's the difference between this and `RESTGetAPIGuildTemplateResult`?

Reply:
different endpoints, but the other one should be an array oops

Reply:
If the data returned is the same, it should.. probably be unified, and exported as an alias

Reply:
you're talking about `export type RESTGetAPITemplateResult = APITemplate;` right? (because this comment is attached to a comment line)
that route returns something else and I edited it, but this route is just the APITemplate object